gradoop-examples/src/main/java/org/gradoop/benchmark/sna/SocialNetworkAnalyticsExample.java

```diff
@@ -136,12 +136,12 @@ public static void main(String[] args) throws Exception {
     }
 
     // group on vertex and edge labels + count grouped edges
-    LogicalGraph groupedGraph  = new Grouping.GroupingBuilder()
+    LogicalGraph groupedGraph  = graph.callForGraph(new Grouping.GroupingBuilder()
```

Comment:
Depending on which PR will be merged first, we need to make this change in the benchmark repo

Reply:
You can also use the API call:
```
graph.groupBy(
      Arrays.asList(Grouping.LABEL_SYMBOL, "name"), 
      Collections.emptyList(), 
      Collections.singletonList(Grouping.LABEL_SYMBOL),
      Collections.singletonList(new Count()), 
      GroupingStrategy.GROUP_COMBINE);
```